ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -1 +1,55 @@
-{}
+type: dict
+allow_other_keys: true
+keys:
+  access_lists:
+    type: list
+    primary_key: name
+    convert_types:
+    - dict
+    display_name: IP Extended Access-Lists
+    description: 'AVD currently supports 2 different data models for extended ACLs:
+
+
+      - The legacy `access_lists` data model, for compatibility with existing deployments
+
+      - The improved `ip_access_lists` data model, for access to more EOS features
+
+
```

Comment:
Extra lines here as compared to fragment

Reply:
Notice the difference in line 10 between ' and |. The yaml formatting for newlines inside strings is just to have two of them... So this is correct.

Reply:
indeed - my bad